Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Name and AssemblyName are required because albacore fails without it -->
+    <Name>Expecto.netcore.Tests</Name>
+    <AssemblyName>Expecto.netcore.Tests</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <DefineConstants>
+      RESHAPED_THREAD_CULTURE;
+      $(DefineConstants)
+    </DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="../Expecto.Tests/Prelude.fs" />
+    <Compile Include="../Expecto.Tests/Tests.fs" />
+    <Compile Include="../Expecto.Tests/SummaryTests.fs" />
+    <Compile Include="../Expecto.Tests/FocusedTests.fs" />
+    <!-- TODO: consider enabling these tests when FsCheck/BenchmarkDotNetTests are ported to netcore
+    <Compile Include="../Expecto.Tests/FsCheckTests.fs" />
+    <Compile Include="../Expecto.Tests/BenchmarkDotNetTests.fs" />
+    -->
+    <Compile Include="../Expecto.Tests/Main.fs" />
+    <None Include="../Expecto.Tests/Tests.fsx" />
+    <None Include="../Expecto.Tests/paket.references" />
+    <Content Include="../Expecto.Tests/app.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
+    <PackageReference Include="FSharp.Core" Version="4.1.0" />
+    <ProjectReference Include="../Expecto.netcore/Expecto.netcore.fsproj" />
+    <PackageReference Include="FsCheck" Version="3.0.0-alpha-b743" />
```

Comment:
@enricosada says the FSharp.Sdk one needs to stay, so that'll have to go in the generator